pandas/core/aggregation.py

```diff
@@ -52,17 +56,17 @@ def normalize_keyword_aggregation(kwargs: dict) -> Tuple[dict, List[str], List[i
 
     Returns
     -------
-    aggspec : dict
+    aggspec : DefaultDict[list]
```

Comment:
What's the reason to use type annotations syntax instead of the docs one we've been using?

Reply:
> What's the reason to use type annotations syntax instead of the docs one we've been using?

I'll change that, forgot that we use a different syntax for the docs